fcs/RELEASE_NOTES.md

```diff
@@ -1,3 +1,6 @@
+#### 23.0.1
```

Comment:
Shouldn't it be something like 28.0.0 already?

Reply:
Yes, changes need to be RI'd back from FCS repository.  I'll ask @baronfel to help with that, thanks

Reply:
I'll take care of that when this merges and I integrate it, so as to not cause conflicts.